cli/config/compose/profiles/fleet/docker-compose.yml

```diff
@@ -33,7 +33,7 @@ services:
     ports:
       - "5601:5601"
     volumes:
-      - ${kibanaConfigPath}:/usr/share/kibana/config/kibana.yml
+      - ./configurations/kibana.config.yml:/usr/share/kibana/config/kibana.yml
```

Comment:
Do we lose any functionality hardcoding the PATH?

Reply:
I'd say no, as there are keys reading env vars, so it's very coupled to the fleet profile